src/WorkflowCore/Models/WorkflowDefinition.cs

```diff
@@ -18,14 +18,16 @@ public class WorkflowDefinition
 
         public WorkflowErrorHandling DefaultErrorBehavior { get; set; }
 
-        public TimeSpan? DefaultErrorRetryInterval { get; set; }                
+        public Type OnPostMiddlewareError { get; set; }
```

Comment:
Does it make sense to define how middleware fails on the workflow level or the global level?
What were the uses cases you had in mind?

Reply:
Basically, as it stands now, you can do either. You can define it at the global level by implementing your own `IWorkflowMiddlewareErrorHandler` or overriding it on the individual workflow level by using `OnPostMiddlewareError`. 

One potential use case I can think of is if your workflow middleware does something like shipping workflow metrics to a timeseries DB like [InfluxDB](https://docs.influxdata.com/influxdb/v1.8/). If for whatever reason the connection to influx is down, you may want to add some special handling for the workflows like queueing up the metrics to ship later on when it comes back up.

Reply:
I was just wondering if it makes sense to be able to define it on the individual workflow level?  What uses cases does that enable?

Reply:
Ah good point. None off the top of my head that couldn't be handled with a global handler. Should I remove it?

Reply:
Yeah, I think so... if we find use cases for it we can always add it but let's try manage the complexity for now.

Reply:
Sounds good. Removed it and updated docs, examples, and tests.